driver-async/src/main/com/mongodb/async/client/SubscriptionHelpers.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2015 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.async.client;
+
+import com.mongodb.Block;
+import com.mongodb.async.SingleResultCallback;
+
+import java.util.List;
+
+
+/**
+ * Subscription helpers.
+ * <p/>
+ * <p>Allows async methods to be converted into event-based style observables.</p>
+ *
+ * @since 3.1
+ */
+public final class SubscriptionHelpers {
```

Comment:
None of these methods are under test, so probably needs a SubscriptionsSpecification of some sort